Yank/tests/test_yaml.py

```diff
@@ -1751,21 +1752,21 @@ def test_run_solvation_experiment():
     """Test solvation free energy experiment run."""
     with omt.utils.temporary_directory() as tmp_dir:
         yaml_script = get_template_script(tmp_dir)
+        yaml_script['solvents']['PME']['clearance'] = '14*angstroms'
```

Comment:
Is `clearance` the size of the box buffer when constructing a box, or the size of the cutoff used for post-processing?